<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>renameDialog</class>
 <widget class="QDialog" name="renameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="enableRandomRadioButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>不再出现,默认截图使用随机命名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sureSaveButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图片名字:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userRenameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>不输入则随机命名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>tips:图片会默认存放在程序的image目录下</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>存放目录:(以下为暂时修改)</string>
   </property>
  </widget>
  <widget class="QLabel" name="dirLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>46</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="changeDirButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>更改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
